lib/system.g

```diff
@@ -111,7 +111,11 @@ BIND_GLOBAL( "GAPInfo", rec(
            help := [ "Run ProfileLineByLine(<filename>) with recordMem := true on GAP start"] ),
       rec( long := "cover", default := "", arg := "<file>",
            help := [ "Run CoverageLineByLine(<filename>) on GAP start"] ),
-      rec( long := "enableMemCheck", default := false)
+      rec( long := "enableMemCheck", default := false),
+      rec( long := "norepl", default := false,
+           help := [ "Disable the GAP REPL" ] ),
```

Comment:
There are people who don't know what a REPL is. There is also no help under `?REPL`. Of course one can google, but I'd prefer if we did not force people to do that. Can we perhaps expand this, e.g. by writing "Disable the GAP read-eval-print loop (REPL)" or so?